syntax = "proto3";

package Packet;

message PlayersListPacket {
	message Player {
		string nickname = 1;
		uint32 position_x = 2;
		uint32 position_y = 3;
		string avatar_name = 4;
	}

	uint32 player_id = 1;
	repeated Player players = 2;

}